xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -39,48 +39,63 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-**$services.localization.render('xe.scheduler.job.name')**
-
-$!doc.display('jobName')
-
-**$services.localization.render('xe.scheduler.job.description')**
-
-$!doc.display('jobDescription')
-
-**$services.localization.render('xe.scheduler.job.expression')**
-
-$!doc.display('cron')
-
-**$services.localization.render('xe.scheduler.job.script')**
-
-#if($xcontext.action=="edit")
-$!doc.display("script")
-#elseif($xcontext.action=="view" &amp;&amp; "$!job.getProperty('script').value" !="")
-{{code language="groovy"}}
-$job.getProperty('script').value
-{{/code}}
+  <content>{{velocity output='false'}}
+#macro (displayJobProperty $obj $propName)
+  #set ($class = $obj.xWikiClass)
+  #set ($prettyName = $class.get($propName).translatedPrettyName)
+  &lt;dt&gt;
+    &lt;label for="${class.getName()}_${obj.number}_${propName}"&gt;$prettyName&lt;/label&gt;
+  &lt;/dt&gt;
+  &lt;dd&gt;
+    $!doc.display($propName, $obj)
+  &lt;/dd&gt;
 #end
+{{/velocity}}
 
-#if($xcontext.action=="edit")
-{{info}}$services.localization.render('xe.scheduler.job.scriptexplanation'){{/info}}
+{{velocity}}
+#if ($doc.getObject('XWiki.SchedulerJobClass'))
+  #set ($job = $doc.getObject("XWiki.SchedulerJobClass"))
+  {{html wiki='true'}}
+  &lt;div class = 'xform'&gt;
+    &lt;dl&gt;
+      #if ($job)
+        #displayJobProperty($job 'jobName')
+        #displayJobProperty($job 'jobDescription')
+        #displayJobProperty($job 'cron')
+      #end
```

Comment:
Should this #if close here?

Reply:
Actually, I just realised that we don't need it at all because it is already checked here:
`#if ($doc.getObject('XWiki.SchedulerJobClass'))
   #set ($job = $doc.getObject("XWiki.SchedulerJobClass"))`

Reply:
Good catch! Could you update the PR?